.github/workflows/codequality.yml

```diff
@@ -0,0 +1,27 @@
+name: Code quality
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Run unit tests
+        run: ./gradlew test
+
+      - name: Run detekt
+        run: ./gradlew detekt
+
+      - name: Run android lint
+        run: ./gradlew lint
+
+      - name: Build project
+        run: ./gradlew build
```

Comment:
doesn't build runs lint as well? if so then maybe assemble is more suitable?

Reply:
i agree, it would be better